logstash-core/logstash-core.gemspec

```
@@ -53,10 +53,7 @@ Gem::Specification.new do |gem|
   gem.add_runtime_dependency "filesize", "0.0.4" #(MIT license) for :bytes config validator
   gem.add_runtime_dependency "gems", "~> 0.8.3"  #(MIT license)
   gem.add_runtime_dependency "concurrent-ruby", "~> 1.0", ">= 1.0.5"
-
-  # Later versions are ruby 2.0 only. We should remove the rack dep once we support 9k
-  gem.add_runtime_dependency "rack", '1.6.6'
-
+  gem.add_runtime_dependency "rack", '>= 1.6.11'
```

This doesn't prevent 2.0.0 from being installed, so maybe instead:

```suggestion
  gem.add_runtime_dependency "rack", ['>= 1.6.11', '~> 1']
```

---

damn ... good catch. I meant to write `~> 1.6.11` but actually `~> 1.6` would be better if there is ever a 1.7 version, which is very unlikely. To make sure 1.6.11 is the minimum I guess your suggestion makes sense, but it should be `['>= 1.6.11', '~> 1.6']`

---

if we don't wish to support 1.7.x then the simplest is `'~> 1.6.11'`

---

I'm good with supporting an unlikely 1.7 ...